src/spec/test/CoercionTest.groovy

```diff
@@ -214,7 +214,7 @@ boolean doFilter(String s) { s.contains('G') }
 Predicate filter = this.&doFilter
 assert filter.accept('Groovy') == true
 
-Greeter greeter = GroovySystem.&getVersion
+Greeter greeter = new GroovySystem().&getVersion
```

Comment:
this change does not make sense to me. getVersion is a static method, so GroovySystem.&getVersion must be fine